Source/WebInspectorUI/UserInterface/Views/Layers3DContentView.js

```diff
@@ -417,6 +523,52 @@ WI.Layers3DContentView = class Layers3DContentView extends WI.ContentView
         WI.layerTreeManager.showPaintRects = !WI.layerTreeManager.showPaintRects;
     }
 
+    _handleShowLayerContentButtonClicked(event)
+    {
+        this._showLayerContent = !this._showLayerContent;
+        this._showLayerContentButtonNavigationItem.activated = this._showLayerContent;
+
+        for (let loadContext of this._pendingTextureLoads.values())
+            loadContext.cancelled = true;
+        this._pendingTextureLoads.clear();
+
+        this._refreshAllLayers();
+    }
+
+    _handleRefreshLayersButtonClicked(event)
+    {
+        for (let loadContext of this._pendingTextureLoads.values())
+            loadContext.cancelled = true;
+        this._pendingTextureLoads.clear();
+
+        this._refreshAllLayers();
+    }
+
+    _refreshAllLayers()
+    {
+        for (let [layerId, layerGroup] of this._layerGroupsById) {
+            this._scene.remove(layerGroup);
+            for (let child of layerGroup.children) {
+                if (child.geometry)
+                    child.geometry.dispose();
+                if (child.material) {
+                    if (child.material.map)
+                        child.material.map.dispose();
+                    child.material.dispose();
+                }
+            }
+        }
+        this._layerGroupsById.clear();
+
+        const zInterval = 25;
+        this._layers.forEach((layer, index) => {
+            let layerGroup = this._createLayerGroup(layer);
```

Comment:
is it really necessary to completely destroy and recreate every single group when refreshing the content of all layers?  could we instead just call `_loadLayerTexture` for each of these to just recreate the texture for the content?